lib/class-wp-theme-json-gutenberg.php

```diff
@@ -485,10 +485,10 @@ class WP_Theme_JSON_Gutenberg {
 	 */
 	const VALID_STYLES = array(
 		'background' => array(
-			'backgroundImage'    => 'top',
-			'backgroundPosition' => 'top',
-			'backgroundRepeat'   => 'top',
-			'backgroundSize'     => 'top',
+			'backgroundImage'    => null,
+			'backgroundPosition' => null,
+			'backgroundRepeat'   => null,
+			'backgroundSize'     => null,
```

Comment:
TODO

- add tests class-wp-theme-json-test.php
- double check that only blocks that have support for background image are included